server/src/main/java/org/elasticsearch/action/admin/indices/create/TransportCreatePartitionsAction.java

```diff
@@ -340,8 +346,6 @@ private Settings createIndexSettings(ClusterState currentState, List<IndexTempla
         if (indexSettingsBuilder.get(IndexMetadata.SETTING_CREATION_DATE) == null) {
             indexSettingsBuilder.put(IndexMetadata.SETTING_CREATION_DATE, new DateTime(DateTimeZone.UTC).getMillis());
         }
-        indexSettingsBuilder.put(IndexMetadata.SETTING_INDEX_UUID, UUIDs.randomBase64UUID());
-
```

Comment:
Interesting that we used to have same UUID when this optimization was introduced for the first time.

https://github.com/crate/crate/commit/9e12f8142debd84b7bc299ddeda84ab19332d9d7#diff-412bbc8f646dc4a7ba56f15bb144be0cd1e672aa296e0b1451b763728b8d5ebfR440